extensions/SemanticResultFormats/formats/Exhibit/exhibit/extensions/time/time-extension.js

```diff
@@ -45,7 +45,7 @@
 
     if (!("Timeline" in window)) {
         //scriptURLs.push("http://api.simile-widgets.org/timeline/2.3.1/timeline-api.js?bundle=true");
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this commented out code. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3ACommentedCode)

[project-id]: wikia:app